### DISCUSSION TITLE
Pediatric data

### DISCUSSION THREAD

**Discussion by kguds** [synthetichealth/synthea#1300] [Ideas]
Hi there
I'm looking for pediatric specific data with a focus on growth charts, realistic immunization history, age-appropriate vitals, weight-based dosing, etc. 
This may be already addressed somewhere in a module or the documentation, I only recently heard about this project and love what I see already.
Cheers!

**Comment by jawalonoski** [comment]
Generally when you run the software it will generate a proportion of population as children (in proportion to the census data). You can override this using the `-a` flag (e.g., adding the `-a 0-10` flag to the command line will only generate patients between 0 and 10 years old).

Pediatric patients will have realistic birth length and weights, growth curves (height and weight), realistic pediatric immunization history, medical history, etc.

Age-appropriate vital signs may be hit or miss. Weight-based dosing is absent.

**Comment by kguds** [comment]
Thanks for the quick reply, sounds like most of what I need then, I'll give it a try!

**Reply by garyisaac** [reply]
Just seeing this post now. I wrote an explanation for pediatric test data for our certification program. See attached. Basically the explanation is the same as this thread. You can use the age parameter.
[Using Synthea to create pediatric test patients.pdf](https://github.com/synthetichealth/synthea/files/12196180/Using.Synthea.to.create.pediatric.test.patients.pdf)
